### DISCUSSION TITLE
Mail server stop sending mail

### DISCUSSION THREAD

**Discussion by jossybenson** [postalserver/postal#1685] [Help with using Postal]
Hello Willpower I still have this issue of my postal server not sending mails with mailing softwares but will send test mail successfully with a testing site like smtper.net, let me attached a screen shot here
![Capture](https://user-images.githubusercontent.com/92524219/139956016-a089ee9d-836f-47c5-b6a8-983b45f3e7b4.PNG)
this is my result if I want to send emails with a mailing software like morphy mail or Gammadyne mailer.
![Capture 2](https://user-images.githubusercontent.com/92524219/139956189-797bfc42-eb0b-4f27-a6f6-fc8c7f3d12eb.PNG)
here a test mail will successfully send and I will receive it on my mail box, but when I want to send email no way please help.

**Comment by willpower232** [comment]
if it works with smtper.net and not your local computer then there must be something on your internet connection stopping it. Also it looks like you're using DigitalOcean and they may be blocking some or all email connections so you could double check with them.

**Reply by jossybenson** [reply]
Yes Willpower am using DigitalOcean.....could it be the cause or my internet connection? I have switch off my pc firewall and try still thesame problem, it can send mail via smtper.net but will never send mail no matter the kind of mailing software I use.

**Reply by pjv** [reply]
Turning off your PC firewall won't help you if your Internet service provider is blocking outgoing traffic on port 25 which many do to combat spam.

If you can get on the command line on your PC, try to telnet to your postal server on port 25:

`telnet your.postal.server 25`

if it can't connect and your PC firewall is turned off, then your ISP is likely blocking outgoing traffic on port 25.
